### PR TITLE
Bump version with updatecli

### PR DESCRIPTION
**Pull request by reakaleek**
## Details

Move bump opbeans-ruby from elastic/apm-agent-ruby to the consumer

## How to test 

```shell
GIT_USER=ok \
GIT_EMAIL=ok@ok.com \
GITHUB_TOKEN=<personal-access-token>
updatecli -c .ci/bump-opbeans-ruby.yml diff
```

## Related Issues

- https://github.com/elastic/apm-agent-ruby/pull/1366 (The opbeans bump part will be removed there)

### CODE REVIEW

File: .github/workflows/bump-opbeans-ruby.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: ./.ci/bump-opbeans-node.yml
```
Comment (v1v) [inline .github/workflows/bump-opbeans-ruby.yml:24]:
```suggestion
          pipeline: ./.ci/bump-opbeans-ruby.yml
```

Comment (reakaleek) [reply .github/workflows/bump-opbeans-ruby.yml:24]: Aw, thanks 😅

File: .ci/bump-opbeans-ruby.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        org\.opencontainers\.image\.version="(\d+.\d+.\d+)"
+      replacepattern: >-
+        org.opencontainers.image.version="{{ source `elastic-apm-agent-ruby` }}"
+  gem:
+    name: Bump elastic-apm
+    sourceid: elastic-apm-agent-ruby
+    scmid: githubConfig
+    kind: shell
+    spec:
+      command: ./.ci/bump-gem.sh
```
Comment (v1v) [inline .ci/bump-opbeans-ruby.yml:78]: Question:

I wonder whether this step could be replaced by:
- bumping the version in the Gemfile directly
- then run a new target in charge of:
  -  bundle update elastic-apm"
- configure the  `.github/workjflows/bump-opbeans-ruby.yml` to run within a ruby context https://github.com/ruby/setup-ruby

Comment (reakaleek) [reply .ci/bump-opbeans-ruby.yml:78]: I actually had something similar. but I wasn't 100% sure about it.

I was not able to find anything in the documentation, but do you know if I can mark a target dependent on another target in updatecli?

Comment (v1v) [reply .ci/bump-opbeans-ruby.yml:78]: I think targets in the updatecli run sequentially

`dependson` is available for targets -> https://www.updatecli.io/docs/core/target/

Comment (reakaleek) [reply .ci/bump-opbeans-ruby.yml]: Thank you.. I knew I saw it somewhere, but couldn't find it anymore.

https://github.com/elastic/opbeans-ruby/pull/171/commits/cdfa3183077144cc685873e6ab3ba7fe1f8cd3c4

Comment (reakaleek) [reply .ci/bump-opbeans-ruby.yml]: https://github.com/elastic/opbeans-ruby/pull/171/files/bec01713d36fa013fd1174dff276b4de5d336e8b..af3f812222f4add1dda53fba11c37032e704dc34

**Comment by v1v** [comment]
@estolfo , would you mind reviewing it please?

**Pull request merged**